registry/notifications.md

```diff
@@ -29,15 +29,17 @@ order is not guaranteed.
 To setup a registry instance to send notifications to endpoints, one must add
 them to the configuration. A simple example follows:
 
-      notifications:
-        endpoints:
-          - name: alistener
-            url: https://mylistener.example.com/event
-            headers:
-              Authorization: [Bearer <your token, if needed>]
-            timeout: 500ms
+```yaml
+notifications:
+ endpoints:
```

Comment:
looks like this is indented with 1 space instead of two (and everything under it), otherwise the yaml won't be valid